src/app/problem/saveChangeButton/SaveChangeButton.js

```diff
@@ -0,0 +1,11 @@
+import {EditorButton} from "../../commons/ProblemEditorPage/ProblemEditorPage";
+
+function SaveChangeButton () {
+    return (
+        <EditorButton
+            text={"Save Change"} buttonColor={"#96D745"} fontColor={"#FFFFFF"}
+        />
+    )
```

Comment:
you don't need this component since it's just another components.


too many components will cause some inconvenience when we are actually going to **function them**. Think, how do you do when you have to call the backend's API on button click?